### PR TITLE
Getting Started Instructions for WinForms on top of .NET Core 3.0. 

### PR DESCRIPTION
**Pull request by diverdan92**
Adding getting started instructions for WinForms on top of .NET Core 3.0 and reformatting the samples to include a more helpful description

### CODE REVIEW

File: windowsforms/README.md [hunk 1]
```diff
@@ -1,9 +1,88 @@
...
+Visual Studio does not yet support designers and custom tools for .NET desktop development. You can keep your files in the original project and link the generated files to the new project by using the link attribute in the project elements, e.g. `<Compile Link="" />`. See the [sample](helloworld-sharedsource) in this repo for an example of this.
+
+**Migrating WCF Clients**
+
+.NET Core has its own implementation of `System.ServiceModel` with some differences:
+* It's available as NuGet packages (also included in the Windows Compatiblity Pack)
+* Review if your application uses some of the [unsupported features](https://github.com/dotnet/wcf/blob/master/release-notes/SupportedFeatures-v2.1.0.md).
+* If you want to reuse the ServiceReference created by Visual Studio you might get the error `System.PlatformNotSupportedException: 'Configuration files are not supported.'`. This error requires a code change to specify the binding and endpoint address in the service client constructor.
+ 
+## Known Issues
```
Comment (diverdan92) [inline windowsforms/README.md]: Can we add Issues to the samples repo to allow people testing these scenarios to file and track bugs with WPF/WinForms on .NET Core 3? Or will there be too much dupe with the corefx repo?

File: windowsforms/README.md [hunk 2]
```diff
@@ -1,9 +1,88 @@
...
+* [Blog: .NET Core 3 and Support for Windows Desktop Applications](https://blogs.msdn.microsoft.com/dotnet/2018/05/07/net-core-3-and-support-for-windows-desktop-applications/)
+* [Video: Modernizing Desktop Apps on Windows 10 with .NET Core 3.0 and much more](https://channel9.msdn.com/events/Build/2018/BRK3501?term=scott%20hunter&pubDate=year&lang-en=true)
 
-Note: Windows Forms is supported on [.NET Core 3.0](https://github.com/dotnet/core-sdk#installers-and-binaries) and later.
+## Quality disclaimer
+.NET Core 3 support for desktop development is not yet in preview. There are early daily builds available supporting WinForms and WPF. You will likely encounter missing tools, bugs, and unexpected behavior. We do not recommend using this SDK and tools for building applications for production scenarios. We do recommend using this SDK and tools to evaluate your how easy it will be to migrate your existing applications, or if you're just interested in trying out the latest upcoming Windows development technology.
 
-## Samples
+## Samples in this repo
+| Sample Name | Descirption |
```
Comment (richlander) [inline windowsforms/README.md]: Spelling mistake with "Descirption "

File: windowsforms/README.md (hunk 1 above)
Comment (richlander) [reply windowsforms/README.md]: Good q. They should probably use the dotnet/core repo. People should file issues on the samples repo for issues with the samples themselves (or for requests for new samples).

File: windowsforms/README.md (hunk 2 above)
Comment (diverdan92) [reply windowsforms/README.md]: Fixed

**Pull request merged**